recipe/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "kaldi-native-fbank" %}
+{% set version = "1.17" %}
+
+# version is in format 1.X_Y_ZZZZZZ because the release is based on 1.17 from the main developer
+# but a commit was added from a fork from developer cbalioglu
+package:
+  name: {{ name|lower }}
+  version: {{ version }}+1_740f131
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b877a6aea7b09f87dfefd9e01106f38221287a2369fcda9221beb6bb76476d5c
+  patches:
+    - 0001-unvendor-gtest-and-pybind11.patch
+    - 0002-install-cmake-config-for-package.patch
+    - 0003-Update-ExtractWindow-signature.patch
+
+build:
+  # not a must-have, there is a problem with cmake -GNinka not picking up ninja
+  skip: True  # [win]
+  number: 0
+  ignore_run_exports:
+    - gtest
+  missing_dso_whitelist:
+    - $RPATH/ld64.so.*  # [s390x]
+    - $RPATH/libkaldi-native-fbank-core.dylib  # [osx]
```

Comment:
This is concerning.

Reply:
Out of curiosity, how come?

Reply:
the reason is that the `RPATH` is not set correctly during the `overlinking-error` check.
The path is however fixed later when the package is tarballed and the dylib is included in it.

Reply:
>The path is however fixed later when the package is tarballed and the dylib is included in it.

Can you clarify what you mean by fixed? If you mean that it just works, then it's still concerning. conda-build is rarely wrong when it comes to missing DSOs.

Reply:
I got rid of this line. I think I misremembered what happened here.